test_suites/test_ovirt_hostedengine.py

```diff
@@ -15,6 +15,25 @@ def test_guide_link(self):
         self.assert_element_visible(self.GETTING_START_LINK)
         self.assert_element_visible(self.MORE_INFORMATION_LINK)
 
+    def test_default_VM_ENGINE_CONFIG(self):
```

Comment:
There is no assertion in this test, how to decide whether this test is fail or pass?